src/HIDGogoExtension.java

```diff
@@ -242,9 +237,9 @@ private void bootHIDDaemon() throws ExtensionException {
     try {
       String classpath =
         new File(gogoExtensionPath + "gogo.jar").getCanonicalPath() + File.pathSeparator +
-        new File(gogoExtensionPath + "hid4java-0.7.0.jar").getCanonicalPath() + File.pathSeparator +
+        new File(gogoExtensionPath + "lib/hid4java-develop-SNAPSHOT.jar").getCanonicalPath() + File.pathSeparator +
```

Comment:
With the change for the `netLogoPackageExtras` setting, I think this should reference the copy in the root of the folder, so no `lib/`.